### ISSUE TITLE
tintin - selected menu item color is gray

### ISSUE THREAD

**Issue by Allen-B1** [pebble-dev/RebbleOS#56]
When a menu item is selected it is gray but on the original watch firmware it is black

**Comment by jwise** [comment]
I agree!  I think the menus on B&W devices could definitely use improvement!
